content/docs/SUSHI/running/_index.md

```diff
@@ -91,27 +97,19 @@ customized-ig
 │   │   ├── mySecondPage.md
 │   │   ├── myThirdPage.md
 │   │   └── myFourthPage.md
-│   ├── profiles
```

Comment:
Do you think it is worthwhile to include a `profiles` folder in the `input` folder and have it contain an additional StructureDefinition, similar to the documentation Chris had written up earlier? Something like 

```
└── profiles
    └── StructureDefinition-ExternalProfile.json
```

Reply:
I think in the previous examples, we didn't have external profiles, but that `profiles` folder was showing where SUSHI generated profiles into, that is why I didn't include it.

Reply:
Yeah I knew we didn't have any external profiles in the previous example, but I was curious if it would be helpful to include. But now I'm realizing that this says it is output from the customized-ig on the Project Structure page, which doesn't have external profiles. So I now vote to ignore my comment.